nautobot/core/views/generic.py

```diff
@@ -1051,7 +1051,7 @@ def post(self, request, **kwargs):
 
             if form.is_valid():
                 logger.debug("Form validation was successful")
-                return self.send_bulk_edit_objects_to_job(request, form, model)
+                return self.send_bulk_edit_objects_to_job(request, self.form, model)
```

Comment:
Still feels weird/off to me - should we be passing `form.cleaned_data` as a parameter here instead of passing the form class and calling `normalize_querydict()` on the request?